phase-5/README.md

```diff
@@ -1,19 +1,19 @@
 # Phase 5
 :::tip Phase 5 starting soon  
-Phase 5 is planned to start on 08 June 2020 11:00 UTC. 
-
-However, to allow all validators to have a change or earning all the tokens by completing the [node updating challenge](update-node.md), we will start counting the precommits from the first block generated after 09 June 2020 00:00 UTC.  
+Phase 5 is planned to start on 10 July 2020 11:00 UTC. 
 :::
 
-Thanks to the latest Morpheus testnet (`morpheus-5000`), we've implemented the possibility for users of doing two new things: 
+Thanks to the latest Morpheus testnet (`morpheus-7000`), we've implemented the possibility for users of doing three new things: 
 
-1. tagging a person inside a post with a multimedia file; 
-2. reporting a post.
+1. creating posts containing hashtags;
+2. tagging a person inside a post with a multimedia file; 
+3. reporting a post.
 
 Inside Phase 5 you will be able to play with those features and earn some DSM by completing the following challenges:
 
 - [Tag a person inside a post (50 Desmos Tokens)](tag-person.md)
```

Comment:
Please also link the hashtag instruction with rewards here.

Reply:
Done :heavy_check_mark: